jetbrains-core/build.gradle.kts

```diff
@@ -1,15 +1,12 @@
 // Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+// Cannot be removed or else it will fail to compile
```

Comment:
old comment still left over, also how did this work? It would not import properly without the (actually unused) import

Reply:
Removed